README.md

```diff
@@ -1,5 +1,8 @@
 ![Emissary Dark Knight - some code just wants to watch the core burn](emissary-knight.png) 
 
+|__Build Status__|[![CodeQL](https://github.com/NationalSecurityAgency/emissary/actions/workflows/codeql-analysis.yml/badge.svg)](https://github.com/NationalSecurityAgency/emissary/actions/workflows/codeql-analysis.yml)|
```

Comment:
Not sure why the table is not formatting correctly

Reply:
```suggestion
|     |    |
| ---- | ---- |
|__Build Status__|[![CodeQL](https://github.com/NationalSecurityAgency/emissary/actions/workflows/codeql-analysis.yml/badge.svg)](https://github.com/NationalSecurityAgency/emissary/actions/workflows/codeql-analysis.yml)|
```

Reply:
you need to add a header to the table, even if a blank one